src/index.js

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import SignIn from './screens/SignIn';
 import BottomTabNav from './screens/BottomTabNav';
+import TopTabNav from './screens/TopTabNav';
+
 
 function App() {
   return(
     //<SignIn/>
```

Comment:
signin page is missing

Reply:
import SignIn from './screens/SignIn';
here is imported Sign In page.